FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -429,13 +429,16 @@ def getInstallations( self, argss ):
         else:
           installedBy = installation[ 'InstalledBy' ]
 
+        if not installation[ 'UnInstalledBy' ]:
+          uninstalledBy = ''
+        else:
+          uninstalledBy = installation[ 'UnInstalledBy' ]
```

Comment:
That would result on an error, as the UnInstalledBy field is always present, it might just contain the None value, which is what is tested in that code.